woeip/apps/air_quality/migrations/0004_auto_20190613_0043.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.7 on 2019-06-13 00:43
```

Comment:
Could the migrations be compressed into one (by running migrate once on a fresh master version of the database)? They are a bit redundant.

Reply:
I agree. Though, I didn't want to invest too much time addressing it; I don't think it significantly hinders performance. 

That said, if you know a quick/straightforward method, then let's do it!

Reply:
Agreed, I don't think it's a big deal since it's just a couple so far. I believe there's a method for squashing them but should be fine to do so at some point later when there are more accumulated!